src/commands/notify.rs

```diff
@@ -142,20 +109,111 @@ pub struct Notify {
 
     /// Max number of retries
     #[arg(short = 'r', long = "retries", default_value = "15")]
-    retries: usize,
+    retries: u8,
 
     /// DNS servers to send packet to
     #[arg(required = true)]
     servers: Vec<String>,
 }
 
+const LDNS_HELP: &str = "\
+usage: ldns-notify [other options] -z zone <servers>
+Ldns notify utility
+
+ Supported options:
+        -z zone         The zone
+        -I <address>    source address to query from
```

Comment:
Maybe we should mention that this is currently unsupported?

Reply:
Good idea!